.github/workflows/diff.yml

```diff
@@ -0,0 +1,118 @@
+name: 'Create ROBOT diffs on Pull requests'
+
+on:
+  # Triggers the workflow on pull request events for the master branch
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  edit_file:
+    runs-on: ubuntu-latest
+    container: obolibrary/odklite:v1.2.32
+    steps:
+      - uses: actions/checkout@v2
+      # Checks-out main branch under "main" directory
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+          path: master
+      - name: Diff classification
+        run: export ROBOT_JAVA_ARGS=-Xmx6G; robot diff --left master/src/ontology/cl-edit.owl --left-catalog master/src/ontology/catalog-v001.xml --right src/ontology/cl-edit.owl --right-catalog src/ontology/catalog-v001.xml -f markdown -o edit-diff.md
+      - name: Upload diff
+        uses: actions/upload-artifact@v2
+        with:
+          name: edit-diff.md
+          path: edit-diff.md
+  classify_pr:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.2.32
+    steps:
+      - uses: actions/checkout@v2
+      - name: Classify ontology
+        run: cd src/ontology; make IMP=FALSE PAT=FALSE MIR=FALSE cl.owl
+      - name: Upload PR cl.owl
+        uses: actions/upload-artifact@v2
+        with:
+          name: cl-pr.owl
+          path: src/ontology/cl.owl
+          retention-days: 1
+  classify_master:
```

Comment:
We avoid `master` nomenclature now for political correctness reasons, use `classify_main` and `classify_branch` instead.